accounts/views.py

```diff
@@ -20,7 +26,49 @@ def post(self, request, format='json'):
         serializer = UserSerializer(data=request.data)
         if serializer.is_valid():
             user = serializer.save()
+            # TODO temporary overloading is_active field as a mean to confirm the email.
+            # A better solution would be implement a profile model and add a is_email_confirmed
+            # field there.
+            user.is_active = False
+            user.save()
             if user:
+                url = reverse('activate_account',
+                              kwargs={
+                                  'uidb64': urlsafe_base64_encode(force_bytes(user.pk)).decode(),
+                                  'token': account_activation_token.make_token(user)
+                              },
+                              request=request)
+                message = f'''Hi {user.username},
+Please click on the link to confirm your registration,
+
+{url}'''
+                send_mail('Confirm your email', message, from_email='localhost', recipient_list=[user.email])
                 return Response(serializer.data, status.HTTP_201_CREATED)
 
         return Response(serializer.errors, status.HTTP_400_BAD_REQUEST)
+
+
+class ActivateAccount(APIView):
+    """API to activate account after signing-up.
+
+    get:
```

Comment:
Please use named type annotation in input parameters, e.g. (input_str: str)

Please follow this style for comments:

"""<function purpose>

Args:
    <arg1>: description
    <arg2>: description

Returns:
    <return type>: description
"""